docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -77,6 +77,15 @@ KUBE_SERVICE_ADDRESSES="--portal_net=10.254.0.0/16"
 KUBE_API_ARGS=""
 ```
 
+* *Optional* Edit /etc/kubernetes/controller-manager and remove --machines=127.0.0.1 from the KUBELET_ADDRESS. Leaving this in won't hurt anything but it will cause the output to note that the 127.0.0.1 node is NotReady because we will not be configuring one in this guide.
```

Comment:
Here you say KUBELET_ADDRESS, below you show KUBELET_ADDRESSES? Are they supposed to be the same?